tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -163,11 +163,11 @@ Spawn Notebook With Arguments  # robocop: disable
          ${is_launcher_selected} =  Run Keyword And Return Status  JupyterLab Launcher Tab Is Selected
          Run Keyword If  not ${is_launcher_selected}  Open JupyterLab Launcher
          Open With JupyterLab Menu  File  New  Notebook
-         Sleep  1
+         Sleep  2
```

Comment:
I think you're modifying this also in #276 to 3s. Keep them both at 2s or 3s, otherwise there will be merge conflicts